### ISSUE TITLE
MCLog not playing nice with CocoaLumberjack

### ISSUE THREAD

**Issue by benf1977** [yuhua-chen/MCLog#21]
I noticed that log statements were not be output to my console a short time after starting my run. In my app delegate's didFinishLaunching I call:

    [DDLog addLogger:[DDASLLogger sharedInstance]];
    [DDLog addLogger:[DDTTYLogger sharedInstance]];

I see a few log statements appear, but then they stop. Perhaps MCLog is changing up the TTY logger at some point after launch? I haven't looked into the source code for either project.

**Comment by abbood** [comment]
same problem here..

**Comment by alexlee002** [comment]
Yes, it's better to support CocoaLumberjack. But I'm busy these months... I'll add this feature to my TO-DO list.

**Comment by woaibujian** [comment]
Also miss the log text color (XcodeColors plugin) randomly.
